### PR TITLE
[@types/dxf] Add missing `width` and `height` definition to Box2

### PR DESCRIPTION
**Pull request by whewchews**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] [Add or edit tests](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#my-package-teststs) to reflect the change.
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] [Run `pnpm test <package to test>`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#running-tests).

Select one of these and delete the others:

If changing an existing definition:
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: [dxf source code](https://github.com/skymakerolof/dxf/commit/66f958ab7fbabb9eaf341ece337271e7d677366d) , [vecks source code](https://github.com/bjnortier/vecks/blob/master/src/Box2.js)

### CODE REVIEW

**Comment by DangerBotOSS** [comment]
## Formatting

The following files are not formatted:
1. types/dxf/Utils.d.ts

Consider running `pnpm dprint fmt` on these files to make review easier.
<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against 9c8ee1def8235d7567e431764aa28afd7ada43da
</p>

File: types/dxf/Utils.d.ts [hunk 1]
```diff
@@ -6,9 +6,12 @@ export interface ParsedContent {
 
 export type UtilVertex = [number, number];
 
+// [ref](https://github.com/bjnortier/vecks/blob/master/src/Box2.js)
 export interface Box {
     min: Common.Point2D;
     max: Common.Point2D;
     valid: boolean;
+    width?: number;
```
Comment (RyanCavanaugh) [inline types/dxf/Utils.d.ts:14]: These don't look optional in the source. Why are they marked `?` ?

**Review by RyanCavanaugh — CHANGES_REQUESTED** [review]
See comment

File: types/dxf/Utils.d.ts (hunk 1 above)
Comment (whewchews) [reply types/dxf/Utils.d.ts]: Thank you for pointing that out. I have removed the optional types as per your suggestion.

**Review by RyanCavanaugh — APPROVED** [review]

**Comment by whewchews** [comment]
Ready to merge

**Pull request merged**